packages/assets-controllers/src/crypto-compare-service/crypto-compare.ts

```diff
@@ -0,0 +1,145 @@
+import { handleFetch } from '@metamask/controller-utils';
+
+import type { Rate, ConversionRates } from '../RatesController/types';
+
+/**
+ * A map from native currency symbol to CryptoCompare identifier.
+ * This is only needed when the values don't match.
+ */
+const nativeSymbolOverrides = new Map([['MNT', 'MANTLE']]);
+
+const CRYPTO_COMPARE_DOMAIN = 'https://min-api.cryptocompare.com';
+
+/**
+ * Get the CryptoCompare API URL for getting the conversion rate from the given native currency to
+ * the given currency. Optionally, the conversion rate from the native currency to USD can also be
+ * included in the response.
+ *
+ * @param currentCurrency - The currency to get a conversion rate for.
+ * @param nativeCurrency - The native currency to convert from.
+ * @param includeUSDRate - Whether or not the native currency to USD conversion rate should be
+ * included in the response as well.
+ * @returns The API URL for getting the conversion rate.
+ */
+function getPricingURL(
+  currentCurrency: string,
+  nativeCurrency: string,
+  includeUSDRate?: boolean,
```

Comment:
Should this be optional?

Reply:
This method was already here. Is better to not change it